packages/gestalt/src/contexts/I18nProvider.js

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+import { type Context, createContext, useContext } from 'react';
+
+type PropTranslations = {|
+  [propName: string]: string,
```

Comment:
Please be more strict around which translations are required for which components